nautobot/docs/user-guide/platform-functionality/savedview.md

```diff
@@ -43,3 +44,123 @@ The `config` attribute holds the configuration of the list view, stored as a `di
+
+### How to update an existing Saved View
+
+Stay on the same page where we just created the new Saved View and make some modifications to the view.
+
+Note that the title font of the saved view became italicized. This is an indicator that this saved view currently has unsaved changes.
+
+![Unsaved saved view](../feature-guides/images/saved-views/unsaved-saved-view.png)
+
+Once you have made desired changes, click the "Saved Views" dropdown button and you should see the option "Update Current Saved View".
+
+Click "Update Current Saved View" option and the page should refresh and you should see the success banner on top of the page stating "Successfully updated current view Campus Staging Location List View".
+
+Congratulations, you have successfully updated your saved view!
+
+![Updated saved view](../feature-guides/images/saved-views/updated-saved-view.png)
+
+### How to navigate Saved Views
+
+There are several ways you can navigate saved views. All saved views created by any user for any list view can be accessed from the navigation menu on the left hand side under "Extensibility -> Saved Views" as an admin or user with "users:view_savedview" permission:
+
+![Navigation Menu](../feature-guides/images/saved-views/navigation-menu.png)
+
+Another way is from the "Saved View" dropdown button on any given model list view. All saved views defined for this specific model list view are populated in the dropdown menu and the saved view the user is currently viewing will be highlighted in bold.
+
+![Location List View with Saved Views](../feature-guides/images/saved-views/location-list-view-with-saved-views.png)
+
+![Current Saved View](../feature-guides/images/saved-views/current-saved-view-drop-down-menu.png)
+
+### How to set a saved view as global default for all users
+
+There can only be one global default for each object list view and the only way to set a saved view as the global default for all users is from the SavedViewTable at SavedViewListView that is only accessible to admins and users with "users:view_savedview" and "users:edit_savedview" permissions.
+
+![Saved View Table Edit Button](../feature-guides/images/saved-views/saved-view-admin-edit-buttons.png)
+
+Clicking on the edit button will direct you to a standard edit view where you can edit the `name`, `is_shared` and `is_global_default` attributes of the current saved view.
+
+![Saved View Edit View](../feature-guides/images/saved-views/saved-view-admin-edit-view.png)
+
+Check the box for `is_global_default` and click on the Update button.
+
+You should be redirected to the SavedViewListView with a success message. Now all users without a default view of their own will be directed to "Campus Staing Location List View" when they are trying to access location list view.
+
+![Saved View Edit Success](../feature-guides/images/saved-views/saved-view-admin-edit-success.png)
+
+### How to hide a saved view from all other users but yourself
```

Comment:
I'd like to invert this documentation (see also comment below) and make it so that we default to creating a non-shared saved view and this section describes how to share it with other users. Let's also avoid the term "private" here as non-shared saved views are not truly private - it's a UI feature, not a security feature.

Reply:
Hmmm, I am of the opinion that most users would want to share their saved views with other users. And making a non-shared saved view would be a rare case IMO and it would make sense to come later in the documentation. I do agree I should avoid using the term "private".